### PR TITLE
Feat/window protocol

### PR DESCRIPTION
**Pull request by adica**

### CODE REVIEW

File: dist/xdomain_cookie.js [hunk 1]
```diff
@@ -68,223 +68,228 @@
...
-	exports.xDomainCookie = xDomainCookie;
+(function (exports) {
+    "use strict";
+
+    var xDomainCookie = function (iframe_path, namespace, xdomain_only, iframe_load_timeout_ms, secure_only, debug) {
+        //iframe_path = full TLD (and optional path) to location where iframe_shared_cookie.html is served from, and domain cookie will be set on
+        //namespace = namespace to use when identifying that postMessage calls incoming are for our use
+
+        if (iframe_path.substr(0, 2) === '//') {
+            //verify protocol is present & used
```
Comment (pb-nati) [inline dist/xdomain_cookie.js]: - [ ] Use the parent only if you see 'about:blank'. otherwise, leave it as is.

Comment (adica) [reply dist/xdomain_cookie.js]: it was 'about:' (without the 'blank')

Comment (pb-nati) [reply dist/xdomain_cookie.js]: then test for about..

**Comment by adica** [comment]
@pb-nati

**Pull request merged**